### PR TITLE
Ignore empty schema string (fix #72)

### LINKED ISSUE #72: Avoid empty string in query when string.Empty is specified as default schema
We have generated context with following configuration in `OnModelCreating:`

`modelBuilder.HasDefaultSchema(String.Empty);`

When running queries, the context generates (please note the double quotes for the schema name)

`SELECT "Extent1"."<column1>", "Extent1"."<column2>",...
FROM ""."<table>" AS "Extent1" WHERE ...
`

while same context configuration on Oracle produces

`SELECT "Extent1"."<column1>", "Extent1"."<column2>",...
FROM "<table>" AS "Extent1" WHERE ...
`

Is it possible to avoid the schema in the query string when schema name is an empty string?

### PR DESCRIPTION
**Pull request by superfuego**
This is for issue #72 and should allow our company to switch over to Npgsql from Devart

### CODE REVIEW

**Comment by roji** [comment]
@Emill, @rwasef1830, can you review?

**Comment by shawnphoffman** [comment]
Any update on this?

**Comment by rwasef1830** [comment]
I'll take a look at this in a few days, been very busy as of late.

**Comment by rwasef1830** [comment]
This is missing unit tests that reproduce the problem and prove that your fix works. @superfuego If you're no longer interested in working on this, I can take over if you want.

File: src/EntityFramework6.Npgsql/SqlGenerators/SqlBaseGenerator.cs [hunk 1]
```diff
@@ -553,12 +553,29 @@ public override VisitedExpression Visit([NotNull] DbScanExpression expression)
             ScanExpression scan;
             var overrideSchema = "http://schemas.microsoft.com/ado/2007/12/edm/EntityStoreSchemaGenerator:Schema";
             if (expression.Target.MetadataProperties.TryGetValue(overrideSchema, false, out metadata) && metadata.Value != null)
-                scan = new ScanExpression(QuoteIdentifier(metadata.Value.ToString()) + "." + QuoteIdentifier(tableName), expression.Target);
+            {
+                var schema = metadata.Value.ToString();
+                if (schema == string.Empty)
+                    scan = new ScanExpression(QuoteIdentifier(tableName), expression.Target);
```
Comment (rwasef1830) [inline src/EntityFramework6.Npgsql/SqlGenerators/SqlBaseGenerator.cs]: Please use braces in your if statements and else statements.

**Comment by superfuego** [comment]
I'm happy for you to take over this.

**Comment by rwasef1830** [comment]
Taking over in #77

**Pull request closed without merging**